src/main/java/com/ing/data/cassandra/jdbc/Utils.java

```diff
@@ -264,6 +269,9 @@ public static Properties parseURL(final String url) throws SQLException {
                 if (params.containsKey(KEY_SSL_ENGINE_FACTORY)) {
                     props.setProperty(TAG_SSL_ENGINE_FACTORY, params.get(KEY_SSL_ENGINE_FACTORY));
                 }
+                if (params.containsKey(KEY_CLOUD_SECURE_CONNEC_BUNDLE) ) {
+                	props.setProperty(TAG_CLOUD_SECURE_CONNEC_BUNDLE, params.get(KEY_CLOUD_SECURE_CONNEC_BUNDLE));
```

Comment:
Please fix indentation and code style.
```suggestion
                if (params.containsKey(KEY_CLOUD_SECURE_CONNECT_BUNDLE)) {
                    props.setProperty(TAG_CLOUD_SECURE_CONNECT_BUNDLE, params.get(KEY_CLOUD_SECURE_CONNECT_BUNDLE));
```